### PR TITLE
GVB-01 & M-05 fixes

### PR DESCRIPTION
**Pull request by dovgopoly**

### CODE REVIEW

File: contracts/gov/validators/GovValidators.sol [hunk 1]
```diff
@@ -45,25 +47,35 @@ contract GovValidators is IGovValidators, OwnableUpgradeable {
...
+
     function __GovValidators_init(
         string calldata name,
         string calldata symbol,
         uint64 duration,
         uint128 quorum,
         address[] calldata validators,
-        uint256[] calldata balances
+        uint256[] calldata balances,
+        address _govAddress
```
Comment (Arvolear) [inline contracts/gov/validators/GovValidators.sol:63]: Naah, onlyOwner will be the gov address

File: test/gov/GovPool.test.js [hunk 2]
```diff
@@ -972,6 +975,22 @@ describe("GovPool", () => {
         it("should revert when try move without vote", async () => {
           await truffleAssert.reverts(govPool.moveProposalToValidators(3), "Gov: can't be moved");
         });
+
+        it("should revert when validators count is zero", async () => {
```
Comment (Arvolear) [inline test/gov/GovPool.test.js:979]: Please check in any test where there is a validators vote that the status becomes "executed" after the execution

File: contracts/gov/validators/GovValidators.sol (hunk 1 above)
Comment (dovgopoly) [reply contracts/gov/validators/GovValidators.sol:63]: done

File: test/gov/GovPool.test.js (hunk 2 above)
Comment (dovgopoly) [reply test/gov/GovPool.test.js:982]: done

**Pull request merged**